best-practices/largescale-deep-learning/Debugging/Compute/README.md

```diff
@@ -0,0 +1,8 @@
+## Run Node Health Checks
+This command job is used to test the compute nodes that are being used for training. During training, issues with the compute nodes can cause problems that may affect the training in unexpected ways. Oftentimes it can be hard to determine if the compute nodes are the source of the issue. The node health checks in this command job test multiple parts of the compute to try and find any issues before training.
```

Comment:
can we also add some use cases and expected outputs in the README? 
and also update Training folder README to have a section on node health check and link to this folder?